src/frontend/src/components/ImportanceSliders.tsx

```diff
@@ -0,0 +1,63 @@
+import { useState } from "react";
```

Comment:
I'm open to suggestions for the name of this component.

Reply:
Looking back in git history these were referred to as "importance" (`crimeImportance`, `schoolsImportance`, etc) so I think this naming is intuitive and keeps the thread 👍